src/goto-instrument/goto_instrument_parse_options.cpp

```diff
@@ -1177,6 +1177,25 @@ void goto_instrument_parse_optionst::instrument_goto_program()
       contracts.apply_loop_contracts(to_exclude_from_nondet_static);
   }
 
+  if(cmdline.isset("synthesize-loop-invariants"))
```

Comment:
Please make sure you also add this to the `help` function at the bottom of the file.